src/cli/cli_srp_client.cpp

```diff
@@ -288,14 +514,66 @@ template <> otError SrpClient::Process<Cmd("service")>(Arg aArgs[])
     otError error = OT_ERROR_NONE;
     bool    isRemove;
 
+    /**
+     * @cli srp client service
+     * @code
+     * srp client service
+     * instance:"ins2", name:"_test2._udp,_sub1,_sub2", state:Registered, port:111, priority:1, weight:1
+     * instance:"ins1", name:"_test1._udp", state:Registered, port:777, priority:0, weight:0
+     * Done
+     * @endcode
+     * @par api_copy
+     * #otSrpClientGetServices
+     */
     if (aArgs[0].IsEmpty())
     {
         OutputServiceList(0, otSrpClientGetServices(GetInstancePtr()));
     }
+    /**
+     * @cli srp client service add
+     * @code
+     * srp client service add ins1 _test1._udp 777
+     * Done
+     * @endcode
+     * @cparam srp client service add @ca{instancename} @ca{servicename} @ca{port} [@ca{priority}] [@ca{weight}]
+     * [@ca{txt}] Use names for the `instancename` and `servicename` parameters that help describe the behavior and
+     * function that the device provides. The 'servicename` paramater can can optionally include a list of service
+     * subtype labels that are separated by commas. The examples here use
+     * generic naming.
```

Comment:
Suggest removing the sentence  

"Use names for the `instancename` and `servicename` parameters that help describe the behavior and function that the device provides"

We don't want to instruct users on how to select/choose the service and instance names. These are defined by other protocols (e.g. matter).